### PR TITLE
scheduler: fix `isHot` in `checkHot`

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

fix `isHot` in `checkHot`

### Check List

Tests

- Unit test

### Release note

```release-note
None
```

### CODE REVIEW

**Review by HunDunDM — COMMENTED** [review]
rest LGTM

**Review by disksing — APPROVED** [review]

File: server/schedulers/hot_region.go [hunk 1]
```diff
@@ -817,7 +817,7 @@ func (bs *balanceSolver) calcProgressiveRank() {
 			dstRate := dstLd.Loads[dim]
 			peerRate := peer.GetLoad(getRegionStatKind(bs.rwTy, dim))
 			decRatio := (dstRate + peerRate) / getSrcDecRate(srcRate, peerRate)
-			isHot := peerRate >= bs.sche.conf.GetMinHotKeyRate()
+			isHot := peerRate >= bs.getMinRate(dim)
```
Comment (nolouch) [inline server/schedulers/hot_region.go:820]: need test?

Comment (lhy1024) [reply server/schedulers/hot_region.go:820]: done

**Review by nolouch — APPROVED** [review]

**Review by HunDunDM — APPROVED** [review]

File: server/schedulers/hot_test.go [hunk 2]
```diff
@@ -409,6 +409,38 @@ func (s *testHotWriteRegionSchedulerSuite) TestUnhealthyStore(c *C) {
 		// no panic
 	}
 }
+func (s *testHotWriteRegionSchedulerSuite) TestCheckHot(c *C) {
```
Comment (rleungx) [inline server/schedulers/hot_test.go:412]: add a space

Comment (lhy1024) [reply server/schedulers/hot_test.go:413]: done

**Pull request merged**